packages/hash/api/src/graphql/resolvers/user/util.ts

```diff
@@ -37,6 +37,41 @@ export const mapUserModelToGQL = async (
   };
 };
 
+type GQLOrgExternalResolvers =
+  | "emailInvitations"
+  | "invitationLinks"
+  | "memberships"
+  | "memberOf"
+  | "entityType"
+  | "linkGroups"
+  | "linkedEntities"
+  | "linkedAggregations";
+
+export type UnresolvedGQLOrg = Omit<GQLOrg, GQLOrgExternalResolvers>;
+
+export const mapOrgModelToGQL = async (
+  org: OrgModel,
+): Promise<UnresolvedGQLOrg> => {
+  return {
+    accountId: org.entityId,
+    id: org.entityId,
+    entityId: org.entityId,
+    entityVersionId: org.version,
+    entityTypeId: "" /** @todo: deprecate this field */,
+    entityTypeVersionId: org.entityTypeModel.schema.$id,
```

Comment:
Compared to the above this seems wrong:
we have `entityId`, which is equal to the ID which remains the same and stores the version separately. Here, we store the Id (URI) and the version in one field and soft-deprecating the Id field. Shouldn't `entityTypeId` be the BaseURI and `entityTypeVersionId` be the version?

Reply:
>  Shouldn't entityTypeId be the BaseURI and entityTypeVersionId be the version?

I think more likely we will want to rework this GQL type entirely so that we have `entityTypeBaseUri`/`entityTypeVersionedUri` or similar as fields instead of `entityTypeId`/`entityTypeVersionId` which are legacy fields?

The reason I've not addressed it here is because it would introduce a substantial diff in the FE. I realise we don't have a task for this though, so [have created one](https://app.asana.com/0/1202805690238892/1202980861294701/f). This will block merging into `main`.